### PR TITLE
Added the class to remove Located Event mark

### PR DESCRIPTION
**Pull request by kummy98**

### CODE REVIEW

File: demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import androidx.fragment.app.DialogFragment;
+
+import com.google.android.gms.maps.model.Marker;
+
+/**
+ * @author Kumar
+ * <p>
+ * This class defines a DialogFragment to remove a Located Event mark
+ */
+public class RemoveLocatedEventDialog extends DialogFragment {
```
Comment (CremaLuca) [inline demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java]: Codacy Issue found: [Class should define a constructor.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4807675)

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4807675&bid=15994009)

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4807675&bid=15994009)

File: demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java (hunk 1 above)
Comment (CremaLuca) [inline demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java]: Codacy Issue found: [Class should define a constructor.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4807675)

Comment (CremaLuca) [reply demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java]: Spe che tolgo sta roba

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4807675&bid=15994009)

File: demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java (hunk 1 above)
Comment (CremaLuca) [inline demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java:18]: Codacy Issue found: [Class should define a constructor.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4807675)

**Review by turcato-niccolo — APPROVED** [review]
👌👌

**Pull request merged**